lhotse/post.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c)  2021  Xiaomi Corp.       (author: Fangjun Kuang)
+
+from dataclasses import dataclass
+
+import numpy as np
+
+from lhotse.features.io import FeaturesWriter, get_reader
+from lhotse.utils import uuid4
+
+
+@dataclass(order=True)
+class Posteriors:
+    """
+    Represent the output of the neural network.
+    It contains metadata about how it's stored: storage_type describes "how to read it", for now
+    it supports numpy arrays serialized with np.save, as well as arrays compressed with lilcom;
+    storage_path is the path to the file on the local filesystem.
+    """
+    num_frames: int
+
+    num_dim: int  # model output dim
```

Comment:
How about one of `num_classes`, `num_hidden`, `num_outputs`, `num_activations`, etc.? `num_dim` suggests to me it's more like `len(shape)`